doc/performance-investigations/0010-web-server-worker-class/summary.md

```diff
@@ -0,0 +1,62 @@
+# Web server worker class: Async vs Threads vs Threads + gRPC
```

Comment:
We have previously tested async vs threads vs threads + gRPC in https://github.com/ONSdigital/eq-survey-runner-benchmark/blob/master/doc/performance-investigations/0008-use-gthread-grpc/summary.md. I think we should present this as a re-run of that investigation now that sessions are in Datastore. By linking to that investigation all of the documentation around the why of threads and gRPC is already covered.

Reply:
Done.